### PR TITLE
BD-2713: Criteo audience sync GA - MERGE 1/12

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2713**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_partners/canvas_steps/overview.md [hunk 1]
```diff
@@ -12,30 +12,48 @@ Tool:
 
 > The Braze Audience Sync feature helps you extend the reach of your campaigns to many of the top social and advertising technologies. Through [Braze Canvas]({{site.baseurl}}/user_guide/engagement_tools/canvas), brands can dynamically and securely sync first-party user data into the advertising ecosystem to drive marketing and operational efficiencies.
 
+{% alert important %}
+This documentation is not intended to provide, nor may it be relied upon as providing legal advice. The use of Audience Sync is subject to specific legal requirements. To ensure that you are using it in compliance with all applicable laws, you should seek the advice of your legal counsel.
```
Comment (dkim32) [inline _docs/_partners/canvas_steps/overview.md:16]: Could we move this to the Data Privacy considerations section within the overview?

File: _docs/_partners/canvas_steps/overview.md [hunk 2]
```diff
@@ -12,30 +12,48 @@ Tool:
...
 ## How it works
 
-To get started, add an Audience Sync step within your Canvas journey. 
+To use Audience Sync to Google or Facebook, connect your ad account by searching for the partner on the **Technology Partners** page.
 
-![][18]{: style="max-width:35%;"} ![][20]{: style="max-width:28%;"}
+![][3]{: style="max-width:35%;"} ![][4]{: style="max-width:35%;"}
 
-Next, select one or multiple Audience Sync destinations to sync to. 
+Keep in mind that Braze will sync users as soon as they enter the Audience Sync step within your Canvas.
```
Comment (dkim32) [inline _docs/_partners/canvas_steps/overview.md:36]: Before this sentence, let's add: https://docs.google.com/document/d/1-ojMXl3tXRev17QMWoPpB42g9ORb6fD3p5JksQgeqRU/edit?disco=AAABD7I8MRM

File: _docs/_partners/canvas_steps/overview.md [hunk 3]
```diff
@@ -12,30 +12,48 @@ Tool:
...
+
+![][6]{: style="max-width:65%;"}
+
+After selecting your Audience Sync Pro destinations, connect your selected partner ad account by clicking on the partner tile.
+
+![][7]{: style="max-width:75%;"}
+
+Next, you can create your Audience Sync step in Canvas using this Audience Sync Pro destination. 
+
+![][9]{: style="max-width:85%;"}
```
Comment (dkim32) [inline _docs/_partners/canvas_steps/overview.md:56]: we should have images 7 and 9 back to back and have the sentence starting with "next, you can create..."

File: _docs/_partners/canvas_steps/overview.md [hunk 4]
```diff
@@ -12,30 +12,48 @@ Tool:
...
 
 ## Data privacy considerations
```
Comment (dkim32) [inline _docs/_partners/canvas_steps/overview.md:59]: Add the legal disclaimer after the header

**Review by dkim32 — CHANGES_REQUESTED** [review]
Left some comments!

**Comment by lydia-xie** [comment]
@cla-bot check

**Review by dkim32 — APPROVED** [review]
lgtm. thanks for the quick fixes!

**Review by bre-fitzgerald — APPROVED** [review]
lgtm!

**Pull request merged**